lib/coherence/schema.ex

```diff
@@ -254,7 +254,8 @@ defmodule Coherence.Schema do
 
         Returns a changeset ready for Repo.update
         """
-        def lock(user, locked_at \\ NaiveDateTime.utc_now()) do
+        def lock(user, locked_at \\ nil) do
```

Comment:
why not use `Config.datetime_module().utc_now()` as a default param?

Reply:
I believe that default parameters are decided at compile time - then all locked users would have the same locked_at date.

Reply:
Maybe I got this wrong - this suggests, that the expression is only evaluated when needed: https://elixir-lang.org/getting-started/modules-and-functions.html#default-arguments

Reply:
yeah, look:

```elixir
defmodule MarsRover do
  def time(foo \\ NaiveDateTime.utc_now()) do
    foo
  end
end

iex [1] > MarsRover.time
~N[2019-06-20 13:58:51.149503]
iex [2] > MarsRover.time
~N[2019-06-20 13:58:52.923541]
iex [3] >
```
Again, thank you! <3